R/maic_anchored.R

```diff
@@ -58,6 +58,7 @@ maic_anchored <- function(weights_object,
                           trt_common,
                           trt_var_ipd = "ARM",
                           trt_var_agd = "ARM",
+                          normalize_weight = FALSE,
```

Comment:
Do we need to pass this through to bootstrap?

Reply:
yes, we would need to, but this is more important to unanchor

Reply:
added